src/xdocs/config_filters.xml

```diff
@@ -970,23 +987,6 @@ public class UserService {
           for more details.
         </p>
       </subsection>
-      <subsection name="Notes" id="SuppressionXpathFilter_Notes">
-        <p>
```

Comment:
Same.

Reply:
Same for this as well:
https://github.com/checkstyle/checkstyle/blob/b7cee495898a0f374ac9c4859f9dd71c0c64f153/src/xdocs/config_filters.xml#L1025